c7n/resolver.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, session_factory, cache):
         self.cache = cache
 
     def resolve(self, uri):
+        contents = self.cache.get(("uri-resolver", uri))
+        if contents is not None:
```

Comment:
Here you could do 
```
if contents:
  return contents
```

If its not none it resolves to true, unless there is ever a boolean of False in there